stdlib/src/base64/_b64encode.mojo

```diff
@@ -268,12 +250,7 @@ fn b64encode_with_buffers(
         ](
             nb_of_elements_to_load
         )
-        store_incomplete_simd(
-            result.unsafe_ptr() + len(result),
-            result_vector_with_equals,
-            nb_of_elements_to_store,
-        )
-        result.size += nb_of_elements_to_store
```

Comment:
Related to [our discussion about PR contents](https://github.com/modularml/mojo/pull/3814#discussion_r1878848233), I see that the changes in this file are a refactoring to use the new `append()` methods we talked about.

If this refactoring was a small part of the PR adding those new `append()` methods, I think that would be okay. Even though techincally that would be a PR both adding a new feature and doing a refactoring in the same PR, that can be okay as long as the overall size is still small. The "do one thing" idea is just a rule of thumb 🙂